core/snapshot/engine.go

```diff
@@ -1111,7 +1111,7 @@ func (e *Engine) Close() error {
 
 func (e *Engine) OnSnapshotIntervalUpdate(ctx context.Context, interval int64) error {
```

Comment:
Can you write a unittest for this change please?

Reply:
does the system test evidence above not prove that it works?

Reply:
It proves it works right now. It does not prove it works tomorrow.

Reply:
If it were a more complicated fix where a unittest was extremely complicated, then providing a screen shot is sufficient, but when a unittest would be maybe 5 lines and trivial to write, then we should be writing them.